src/main/groovy/com/github/benmanes/gradle/versions/reporter/HtmlReporter.groovy

```diff
@@ -0,0 +1,287 @@
+      printStream.println("</table>")
+      printStream.println("<br>")
+    }
+  }
+
+  private def getCurrentRows(Result result) {
+    List<String> rows = new ArrayList<>();
+    // The following dependencies are using the latest milestone version:
+    DependenciesGroup<Dependency> list = result.getCurrent();
+    rows.add("<thead><tr><th>Name</th><th>Group</th><th>URL</th><th>Current Version</th></tr></thead>")
+    for (Dependency item : list.dependencies) {
+      String rowString;
+      String rowStringFmt = "<tr><td>%s</td><td>%s</td><td>%s</td><td>%s</td></tr>"
+      rowString = String.format(rowStringFmt, item.getName(), item.getGroup(),
+        getUrlString(item.getProjectUrl()), getVersionString(item.getGroup(), item.getName(), item.getVersion()))
+      rows.add(rowString)
+    }
+    return rows
+  }
+
+  private def writeExceedLatestFound(printStream, Result result) {
+    SortedSet<DependencyLatest> versions = result.getExceeded().getDependencies();
+    if (!versions.isEmpty()) {
+      // The following dependencies exceed the version found at the '
+      //        + revision + ' revision level:
+      printStream.println("<H2>Exceeded dependencies</H2>")
+      printStream.println("<p>The following dependencies exceed the version found at the ${revision} revision level:<p>")
+      printStream.println("<table class=\"warningInfo\">")
+      getExceededRows(result).each { printStream.println it }
+      printStream.println("</table>")
+      printStream.println("<br>")
+    }
+  }
+
+  private def getExceededRows(Result result) {
+    List<String> rows = new ArrayList<>();
+    // The following dependencies are using the latest milestone version:
+    DependenciesGroup<DependencyLatest> list = result.getExceeded()
+    rows.add("<thead><tr><th>Name</th><th>Group</th><th>URL</th><th>Current Version</th><th>Latest Version</th></tr></thead>")
+    for (DependencyLatest item : list.dependencies) {
+      String rowString;
+      String rowStringFmt = "<tr><td>%s</td><td>%s</td><td>%s</td><td>%s</td><td>%s</td></tr>"
+      rowString = String.format(rowStringFmt, item.getName(), item.getGroup(),
+        getUrlString(item.getProjectUrl()), getVersionString(item.getGroup(), item.getName(), item.getVersion()),
+        getVersionString(item.getGroup(), item.getName(), item.getLatest()))
+      rows.add(rowString)
+    }
+    return rows
+  }
+
+  private def writeUpgrades(printStream, Result result) {
+    SortedSet<DependencyOutdated> versions = result.getOutdated().getDependencies();
+    if (!versions.isEmpty()) {
+      // The following dependencies exceed the version found at the '
+      //        + revision + ' revision level:
+      printStream.println("<H2>Later dependencies</H2>")
+      printStream.println("<p>The following dependencies have later ${revision} versions:<p>")
+      printStream.println("<table class=\"warningInfo\">")
+      getUpgradesRows(result).each { printStream.println it }
```

Comment:
I get a null for the latest version in the html report

<img width="1307" alt="Screen Shot 2020-09-03 at 1 04 10 PM" src="https://user-images.githubusercontent.com/378614/92161829-09267e00-ede6-11ea-982a-508f0da7ee7e.png">
<img width="608" alt="Screen Shot 2020-09-03 at 1 04 26 PM" src="https://user-images.githubusercontent.com/378614/92161835-0af04180-ede6-11ea-9930-4e3d13df5c5d.png">

Reply:
Could you provide me with the xml snippet?

Reply:
I ran the `examples/groovy` build. If I use `gradle dependencyUpdates -DoutputFormatter=html` then it shows correctly, but when I tried `gradle dependencyUpdates -Drevision=release -DoutputFormatter=html` then I get nulls.

```html
<tr><td>guava</td><td>com.google.guava</td><td><a target="_blank" href="https://github.com/google/guava">https://github.com/google/guava</a></td><td>16.0-rc1 <a target="_blank" href="https://mvnrepository.com/artifact/com.google.guava/guava/16.0-rc1">Mvn</a> <a target="_blank" href="https://bintray.com/bintray/jcenter/com.google.guava%3Aguava/16.0-rc1">Bintray</a></td><td>null  </td></tr>
```

Reply:
ok cool. next commit.

Reply:
Oh, I lost the latest version when trying in Caffeine:

<img width="1157" alt="Screen Shot 2020-09-04 at 11 57 18 AM" src="https://user-images.githubusercontent.com/378614/92276434-cf1cb100-eea5-11ea-809d-98a614a04a20.png">